docs/index.asciidoc

```diff
@@ -187,10 +187,10 @@ destination field. Multiple values returned will be pushed onto the
 destination field as an array. As such, multiple matches across
 multiple source fields will produce duplicate entries in the field.
```

Comment:
[id="plugins-{type}s-{plugin}-xpath_resources"]
====== Additional XPath resources

Reply:
@karenzone What do you mean here? Should I add these 2 lines and what would be the result in adding these?

Reply:
Thanks for asking, Andre. :-) That would add a subheading (second line) and an anchor (first link) so we could link to it (if we ever wanted to).  It breaks up the "wall of text" for easier reading. 

So it would look something like this: 
<img width="859" alt="Screen Shot 2021-12-02 at 5 59 00 PM" src="https://user-images.githubusercontent.com/35154725/144516583-a1dae1dd-df0d-4fa1-8f5f-51540751cde2.png">

Reply:
I have added not this one.
Could you check if that's doing what you meant?